src/pages/content-history.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
                                         m("a", {
                                                 title  : "Preview",
                                                 href   : m.route() + "/" + data.version,
-                                                config : m.route
+                                                target : "_blank"
```

Comment:
Let's change it back then, since those are internal links the new window popping open feels weird.